### PR TITLE
add @type/svg-maps__brazil

### PR DESCRIPTION
**Pull request by joaosutel**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/svg-maps__brazil/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Type definitions for @svg-maps/brazil 1.0
+// Project: https://github.com/VictorCazanave/svg-maps/tree/master/packages/brazil
+// Definitions by: Joao Sutel <https://github.com/joaosutel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// tslint:disable-next-line no-single-declare-module
```
Comment (RyanCavanaugh) [inline types/svg-maps__brazil/index.d.ts:6]: Please write this using the standard module format.

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/svg-maps__brazil/index.d.ts (hunk 1 above)
Comment (joaosutel) [reply types/svg-maps__brazil/index.d.ts]: Done

File: types/svg-maps__brazil/svg-maps__brazil-tests.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  label: 'label'
+};
+
+const SVGMapContainerNoOptional: brazil.Map = {
+  viewBox: 'vieweBox',
+  locations: [{
+  path: 'path',
+  id: 'id',
+  }],
+};
```
Comment (peterblazejewicz) [inline types/svg-maps__brazil/svg-maps__brazil-tests.ts:19]:
```suggestion
const SVGMapContainer: brazil.Map = {
    viewBox: 'viewBox',
    locations: [
        {
            path: 'path',
            id: 'id',
            name: 'name',
        },
    ],
    label: 'label',
};

const SVGMapContainerNoOptional: brazil.Map = {
    viewBox: 'vieweBox',
    locations: [
        {
            path: 'path',
            id: 'id',
        },
    ],
};
```
just white-space align with other types (4 spaces)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@joaosutel thx!

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by joaosutel** [comment]
> LGTM!
> @joaosutel thx!

@peterblazejewicz peterblazejewicz
YW!
Can u merge, please? I haven't write access

**Pull request merged**